Content/default/Build.fs

```diff
@@ -1,20 +1,18 @@
-#r "paket: groupref build //"
-#load "./.fake/build.fsx/intellisense.fsx"
-#r "netstandard"
-
 open Fake.Core
 open Fake.DotNet
 open Fake.IO
 open Farmer
 open Farmer.Builders
 
-Target.initEnvironment ()
+let execContext = Context.FakeExecutionContext.Create false "build.fsx" [ ]
```

Comment:
We're passing an empty list for `args` here. I'm surprised that this works correctly to pick a target. Does passing in variables also work? e.g. `dotnet run -- e Setting=value`

Reply:
Not sure - I don't really use passing in variables like this, I usually rely on environment variables